docs/gettinginvolved/infrastructure/release-process.rst

```diff
@@ -70,49 +86,26 @@ Preparing For a Release
 
    The version number must adhere to :ref:`the Flocker version numbering policy <version-numbers>`.
 
-#. Export the version number of the release being created as an environment variable for later use:
-
-   .. prompt:: bash $
-
-      export VERSION=0.1.2
-
 #. Create an issue in JIRA:
 
-   This should be an "Improvement" in the current sprint, with "Release Flocker $VERSION" as the title, and it should be assigned to yourself.
+   This should be an "Improvement" in the current sprint, with "Release Flocker <version-being-released>" as the title, and it should be assigned to yourself.
    The issue does not need a design, so move the issue to the "Coding" state.
 
-#. Create and log in to a new :doc:`Flocker development machine <vagrant>`:
-
-   This uses SSH agent forwarding so that you can push changes to GitHub using the keys from your workstation.
-
-   Add your SSH key to the ``sshd`` agent.
-   Note that the ssh key you use must be linked to your GitHub account.
+#. Create an environment to do a release in:
 
-   .. prompt:: bash $
+   .. prompt:: bash $,(flocker-release)tmp-5234cf9a4c8fad37$,(flocker-release)flocker$ auto
 
-      [ -e "${SSH_AUTH_SOCK}" ] || eval $(ssh-agent)
-      ssh-add $HOME/.ssh/id_rsa
-
-   This copies your local git configuration from ``~/.gitconfig``.
-   If this does not exist, commits made for the release will be associated with the default Vagrant username and email address.
-
-   This copies your local configuration for `S3`_ from ``~/.aws``.
-   If this does not exist, a later step will create it.
-
-   .. prompt:: bash $
-
-      git clone git@github.com:ClusterHQ/flocker.git "flocker-${VERSION}"
-      cd flocker-${VERSION}
-      vagrant up
-      vagrant scp default:/home/vagrant/.bashrc vagrant_bashrc
-      echo export VERSION=${VERSION} >> vagrant_bashrc
-      vagrant scp vagrant_bashrc /home/vagrant/.bashrc
-      if [ -d ~/.aws ]; then vagrant scp "~/.aws" /home/vagrant; fi
-      vagrant ssh -- -A
+      $ mktmpenv --prompt="(flocker-release)"
```

Comment:
I put in the description:

> Remove permanent changes (e.g. use a temporary environment

I think that it means that the Python dependencies, and the files created in the temporary environment (the git clone) are all removed when the environment is deactivated. In the Vagrant box, permanent changes didn't matter, because the box could be destroyed. Here, I wanted there to be as few permanent changes as possible.